MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,572 @@
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## To customize the ColorPalette
+
+You can change the default colors of the `ColorPalette` in toolbar.
+
+{% tabs %}
+
+{% highlight XAML %}
+  
+<imageEditor:SfImageEditor Source="image.png">
+    <imageEditor:SfImageEditor.ToolbarSettings>
+        <imageEditor:ImageEditorToolbarSettings>
+            <imageEditor:ImageEditorToolbarSettings.ColorPalette>
+                <Color>Yellow</Color>
+                <Color>Pink</Color>
+                <Color>Violet</Color>
+            </imageEditor:ImageEditorToolbarSettings.ColorPalette>
+        </imageEditor:ImageEditorToolbarSettings>
+    </imageEditor:SfImageEditor.ToolbarSettings>
+</imageEditor:SfImageEditor>
+      
+{% endhighlight %}
+
+{% highlight C# %}
+
+    SfImageEditor imageEditor = new SfImageEditor();
+    imageEditor.Source = ImageSource.FromFile("image.png");
+    imageEditor.ToolbarSettings.ColorPalette.Add(Colors.Yellow);
+    imageEditor.ToolbarSettings.ColorPalette.Add(Colors.Pink);
+    imageEditor.ToolbarSettings.ColorPalette.Add(Colors.Violet);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+N> Color palette colors are common for Text, Shape and Pen.
+
+## Default Toolbar items
+
+### Toolbar item
+
+You can customize each toolbar item using the `Name` and `View` properties.
+
+### Name
+
+You can get or set the names of built-in toolbar and dynamically added toolbar items using the `Name` property.
+
+{% highlight C# %}
+
+public MainPage()
+{			
+    InitializeComponent();
+    SfImageEditor imageEditor = new SfImageEditor();
+    imageEditor.Source = ImageSource.FromFile("image.png");
+    string itemName = imageEditor.Toolbars[1].ToolbarItems[0].Name;
+}
+
+{% endhighlight %}
+
+N> The following built-in toolbar item names are available in image editor: `Browse`, `Undo`, `Redo`, `ZoomOut`, `ZoomIn`, `Reset`, `Save`, `SaveEdit`, `Crop`, `Shape`, `Text`, `Pen`, `Effects`, `Rotate`, `FlipHorizontal`, `FlipVertical`, `custom`, `original`, `ellipse`, `circle`, `square`, `3:1`, `3:2`, `4:3`, `5:4`, `7:5`, `16:9`, `Rectangle`, `Circle`, `Arrow`, `Line`, `Dotted`, `DoubleArrow`, `DottedArrow`, `DottedDoubleArrow`, `ShapeStrokeThickness`, `ShapeStrokeColor`, `ShapeFill`, `ShapeStroke`, `AddText`, `StrokeThickness`, `Delete`, `FontFamily`, `TextAlignment`, `FontAttributes`, `TextColor`, `Start`, `Center`, `End`, `DefaultText`, `Handlee`, `Pinyon Script`, `Kaushan Script`, `Boogaloo`, `Bold`, `Italic`, `Color`, `ShapeStrokeOpacity`, `PenStrokeOpacity`, `TextColorOpacity`, `PenStrokeThickness`, `PenStrokeColor`, `Back`, `Blur`, `Contrast`, `Sharpen`, `Brightness`, `Saturation`, `Hue`, `Opacity`,  and `Exposure`.
+
+N> You cannot modify the names of existing built-in toolbar items and cannot create toolbar item with these list.
+
+## Get an item from inbuilt top toolbar
+
+The `ImageEditorToolbarItem` is placed on the top of the image editor, and you can customize the header toolbar item using the `View` and `Name` properties:
```

Comment:
The default top toolbar contains 3 groups and each group contains its own items. You can get the ToolbarItem and customize the properties.

Reply:
Updated